emanifest-py/src/emanifest/client.py

```diff
@@ -841,6 +841,24 @@ def delete_manifest(self, mtn: str) -> RcrainfoResponse:
         """
         endpoint = f'{self.base_url}/api/v1/emanifest/manifest/delete/{mtn}'
         return self.__rcra_request('DELETE', endpoint)
+    
+    def quicker_sign_manifest(self, reg: bool = False, **kwargs) -> RcrainfoResponse:
```

Comment:
We don't need to `reg` argument, there's not a regulator version of Quicker Sign.

Should we shorten the function signature to `def sign_manifest(...)`? it's a little less explicit but shorter and I can't imagine there will be a **Quicker-er** sign in the foreseable future.

Reply:
Good point. Let's shorten it. Additional signing functionality would likely be a modification of this anyway.